index.js

```diff
@@ -226,10 +226,11 @@ async function main() {
   await handleSwitchToBranchCommand(sg, sortedTrainBranches, combinedTrainBranch);
 
   console.log(`I've found these partial branches:`);
+  let idx = 0;
```

Comment:
Nit: would be slightly easier to do `sortedTrainBranches.map((b, idx) =>`

Reply:
Oh yeah! Good point. Changes in.